defaults/main.yml

```diff
@@ -36,20 +37,20 @@ postgresql_cluster_reset: false
 postgresql_database_owner: "{{ postgresql_admin_user }}"
 
 # Extensions
-postgresql_ext_install_contrib: no
-postgresql_ext_install_dev_headers: no
-postgresql_ext_install_postgis: no
+postgresql_ext_install_contrib: false
+postgresql_ext_install_dev_headers: false
+postgresql_ext_install_postgis: false
 
-postgresql_ext_postgis_version: "2.5" # be careful: check whether the postgresql/postgis versions work together
+postgresql_ext_postgis_version: "2.5"     # be careful: check whether the postgresql/postgis versions work together
```

Comment:
@timothylytle There is too much noise in this pull request due to non-significant changes in YAML file formatting.
I understand, the goal was to fix yaml linter warnings. 

But, with so many non-critical changes to the files, it is hard to compare our repo to the upstream repo (https://github.com/ANXS/postgresql).

In long term, it will be better to keep the files in the original format as they were stored in the upstream repo. This will simply future pull of changes from the upsteam repo.

I believe, Yaml linter warnings can be suppressed via config dotfile.

Reply:
Understood!   I will work on this too

Reply:
This should be corrected now,  I removed any updates to do with lint errors, and suppressed those lint errors.

Also, in line with this thinking I also updated the fix for ubuntu support to be inline with what the upstream repo did.

Reply:
Thank you. It is much easier to review PR now.
And yes, please revert the changes to trailing spaces.